incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,40 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates all objects of a [specified type](how-to.md/#admin-resources) 
+from ancestor namespaces to descendant namespaces. However, sometimes this is 
+too restrictive, and you need to create exceptions to certain policies. For example:
```

Comment:
Emphasize and bold `***exceptions***` since it's the first time that word is being introduced: ***exceptions***. Note that this is different to the Google doc but matches the other words in this doc that are introduced for the first time.